web/package.json

```diff
@@ -112,6 +112,7 @@
     "lolex": "^6.0.0",
     "moment-locales-webpack-plugin": "^1.2.0",
     "moment-timezone-data-webpack-plugin": "1.5.0",
+    "preload-webpack-plugin": "^3.0.0-beta.4",
```

Comment:
Is there a stable version we can use?

Reply:
the fork you mentioned @Taggotty is working fine. we can switch to that package